src/MSA.BuildingBlocks.CosmosDbMigration/NoSQL/ContainerMigration.cs

```diff
@@ -0,0 +1,26 @@
+﻿using Microsoft.Azure.Cosmos;
+using MSA.BuildingBlocks.CosmosDbMigration.Abstractions;
+using System.Collections.Generic;
+using System.Dynamic;
+using System.Threading.Tasks;
+
+namespace MSA.BuildingBlocks.CosmosDbMigration.NoSQL;
+internal class ContainerMigration(CosmosClient cosmosClient, string databaseId, string containerId) : BaseContainerMigration(cosmosClient, databaseId, containerId)
```

Comment:
`: BaseContainerMigration(cosmosClient, databaseId, containerId)` should be on the new line